src/test/kotlin/com/example/demo/RepositoriesTests.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.demo
+
+import com.example.demo.entities.User
+import com.example.demo.repositories.UserRepository
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.mockito.MockitoAnnotations
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+
+@DataJpaTest
+class RepositoriesTests @Autowired constructor (
+    val entityManager: TestEntityManager,
+    val userRepository: UserRepository
+) {
+
+    @BeforeAll
+    fun prepare() {
+    }
+
+    @Test
+    fun `When findByLogin then return User`() {
```

Comment:
Is not a relevant test, why do you test the third-party library? The library has its own tests to check the internal functionality